public/js/pimcore/object/tags/dateRange.js

```diff
@@ -82,8 +82,8 @@ pimcore.object.tags.dateRange = Class.create(pimcore.object.tags.abstract, {
 
     getLayoutShow: function () {
         this.component = this.getLayoutEdit();
-        this.component.items[0].setReadonly(true);
-        this.component.items[2].setReadonly(true);
+        this.component.items[0].setReadOnly(true);
+        this.component.items[2].setReadOnly(true);
```

Comment:
```suggestion
        this.component.items.items[0].setReadOnly(true);
        this.component.items.items[2].setReadOnly(true);
```

For me `this.component.items` isn't an array. We need to add another `items` to get the right items.

Reply:
You are absolutely right, my mistake, I had it exactly the same with my local, only I had copied the whole thing into the editor incorrectly...